content/docs/usage.md

```diff
@@ -1,70 +1,100 @@
-themselves with [Github Actions](https://help.github.com/en/actions) or
-[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).
+You'll need a GitLab, GitHub, or Bitbucket account to begin. Users may wish to
+familiarize themselves with [Github Actions](https://help.github.com/en/actions)
+or
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration).
 Here, will discuss the GitHub use case.
 
-⚠️ **GitLab users:** Please see our
-[docs about configuring CML with GitLab](https://github.com/iterative/cml/wiki/CML-with-GitLab).
+## GitLab
 
-🪣 **Bitbucket Cloud users** We support you, too-
-[see our docs here](https://github.com/iterative/cml/wiki/CML-with-Bitbucket-Cloud).🪣
-_Bitbucket Server support is in the works._
+Please see our docs on
+[CML with GitLab CI/CD](https://github.com/iterative/cml/wiki/CML-with-GitLab)
+and in particular the
+[personal access token](https://github.com/iterative/cml/wiki/CML-with-GitLab#variables)
+requirement.
 
-The key file in any CML project is `.github/workflows/cml.yaml`.
+## Bitbucket
+
+Please see our docs on
+[CML with Bitbucket Cloud](https://github.com/iterative/cml/wiki/CML-with-Bitbucket-Cloud).
+_Bitbucket Server support estimated to arrive by mid 2021._
+
+## GitHub
+
+The key file in any CML project is `.github/workflows/cml.yaml`:
 
 ```yaml
 name: your-workflow-name
 on: [push]
 jobs:
   run:
-    runs-on: [ubuntu-latest]
-    container: docker://iterativeai/cml:0-dvc2-base1
+    runs-on: ubuntu-latest
+    # optionally use a convenient Ubuntu LTS + CUDA + DVC + CML image
+    # container: docker://iterativeai/cml:0-dvc2-base1-gpu
+    # container: docker://ghcr.io/iterative/cml:0-dvc2-base1-gpu
     steps:
       - uses: actions/checkout@v2
-      - name: cml_run
-        env:
-          REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # may need to setup NodeJS & Python3 on e.g. self-hosted
+      # - uses: actions/setup-node@v2
+      #   with:
+      #     node-version: '12'
+      # - uses: actions/setup-python@v2
+      #   with:
+      #     python-version: '3.x'
+      - uses: iterative/setup-cml@v1
+      - name: Train model
         run: |
-
           # Your ML workflow goes here
           pip install -r requirements.txt
           python train.py
-
-          # Write your CML report
+      - name: Write CML report
+        env:
+          REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Post reports as comments in GitHub PRs
           cat results.txt >> report.md
           cml-send-comment report.md
 ```
 
-### CML Functions
-
-CML provides a number of helper functions to help package outputs from ML
-workflows, such as numeric data and data vizualizations about model performance,
-into a CML report. The library comes pre-installed on our
+We helpfully provide CML and other useful libraries pre-installed on our
 [custom Docker images](/doc/self-hosted-runners#docker-images). In the above
```

Comment:
Not for this pull request, but we should proably consider creating a separate section for [...] images, as per https://github.com/iterative/cml.dev/issues/86#issuecomment-900989038; they have more use cases than self-hosted runners.

Reply:
Moved to https://github.com/iterative/cml.dev/issues/102